libraries/messages/src/mav_message.rs

```diff
@@ -0,0 +1,20 @@
+use mavlink;
```

Comment:
I don't think any of this belongs here. Instead, I would create a `GroundStationComm` struct (or whatever else name that makes sense) inside the main board crate. This struct would contain all the mavlink logic of this file, handle the logic of incrementing the sequence, and contain all the logic to setup the uart.